### PR TITLE
build(docs): Move API model download location from repo root into the docs directory

### PR DESCRIPTION
**Pull request by Josmithr**
Moves the location from `<root>/_api-extractor-temp-<version>/doc-models` to `docs/_doc-models/<version>`.

Also removes `readVersionData.js`, which is unused.

### CODE REVIEW

File: docs/readVersionData.js
Comment (alexvy86) [inline docs/readVersionData.js:1]: Unrelated?

**Review by alexvy86 — APPROVED** [review]
LGTM other than the deleted file which doesn't seem related. If it is, just call it out in the description.

File: docs/readVersionData.js
Comment (Josmithr) [reply docs/readVersionData.js:1]: Oh, yes. Good catch. I'll add a note in the PR description.

**Review by zhenmichael — APPROVED** [review]
LGTM
